modules/oadp-features-plugins-known-issues.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+// oadp-features-plugins-known-issues
+// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc
+
+:_content-type: CONCEPT
+[id="oadp-features-plugins-known-issues_{context}"]
+= OADP plugins known issues
+
+The following section describes known issues in {oadp-first} plugins:
+
+[id="velero-plugin-panic_{context}"]
+== Velero plugin panics on imagestream backup due to a missing secret
+
+The velero plugin panics on imagestream backup. A missing secret causes this panic error.
+
+When the backup and the Backup Storage Location (BSL) are managed outside the scope of the Data Protection Application (DPA), the OADP controller, meaning the DPA reconciliation does not create the relevant `oadp-<BSL Name>-<BSL Provider>-registry-secret`.
+
+When the backup is run, the openshift-velero-plugin panics on the imagestream backup, with a panic error:
```

Comment:
I don't see `openshift-velero-plugin` in the docs Should we spell it out? [OpenShift Velero plugin](
https://docs.openshift.com/container-platform/4.15/backup_and_restore/application_backup_and_restore/oadp-features-plugins.html#default-velero-cloud-provider-plugins)